Refit/targets/refit.targets

```diff
@@ -42,8 +42,8 @@
   </Target>
 
   <Target Name="_RefitMSBuildVersionCheck"
-          Condition=" '$([System.Version]::Parse($(_RefitMSBuildMinVersion)).CompareTo($([System.Version]::Parse($(MSBuildVersion)))))' &gt; '0' "
-     BeforeTargets="ResolveAssemblyReferences;Build;Rebuild" >
+          Condition="!($(MSBuildVersion) >= $(_RefitMSBuildMinVersion))"
```

Comment:
I don't think this is doing what you think it is -- we need to be using comparisons that are aware of version numbers.